newIDE/app/src/UI/LocalFolderPicker/index.js

```diff
@@ -111,6 +122,20 @@ const LocalFolderPicker = ({
               onChange={(event, value) => setTextValue(value)}
               onBlur={onBlur}
             />
+            {type === 'default-workspace' && (
+              <div
+                style={{
+                  marginLeft: '8px',
+                }}
+              >
```

Comment:
I think you could add this button as an end adornment to the TextField

Reply:
I made all suggestions except this one. I don't see clearly if its ok to put into the TextField an external button not related directly to the component TextField itself. And, its a big change. Could you please think of it again and tell me your opinions regarding this?

Thanks in advance!

Reply:
You can see the result [here with an end adornment](https://gdevelop-storybook.s3.amazonaws.com/master/latest/index.html?path=/story/ui-building-blocks-textfield--end-adornment)
It shouldn't be a big change.
See here:
https://github.com/4ian/GDevelop/blob/master/newIDE/app/src/stories/componentStories/TextField.stories.js#L49-L57
It's a related action since it should reset the text field value